src/main/java/games/strategy/engine/lobby/server/userDB/Database.java

```diff
@@ -71,11 +71,11 @@ private static File getDBRoot() {
 
   public static Connection getDerbyConnection() {
     final String url = "jdbc:derby:ta_users;create=true";
-    return getDerbyConnection(url, getDbProps());
+    return getConnection(url, getDbProps());
   }
 
   public static Connection getPostgresConnection() {
-    final Connection connection = getDerbyConnection("jdbc:postgresql://localhost/", getPostgresDbProps());
+    final Connection connection = getConnection("jdbc:postgresql://localhost/ta_users", getPostgresDbProps());
```

Comment:
Seems so, otherwise we would use the default database and/or need to switch DB connection later.

From: https://jdbc.postgresql.org/documentation/head/connect.html

> jdbc:postgresql://host:port/database
database
The database name. The default is to connect to a database with the same name as the user name.